pyproject.toml

```diff
@@ -9,10 +9,12 @@ requires = [
 
 [project]
 name = "rapids-build-backend"
-version = "0.0.1"
+version = "0.0.2"
```

Comment:
We should handle version bumps independently of feature PRs.

Reply:
Yes. I did this so that https://github.com/rapidsai/cuml/pull/5804 would use my branch instead of `0.0.1` from PyPI.